### DISCUSSION TITLE
pve machine run on laptop

### DISCUSSION THREAD

**Discussion by Kuechenfenster** [dougiteixeira/proxmoxve#266] [Feature request] [answered]
Dear proxmoxve-team,
as some of us run the system on a laptop, could you please also incl. the main machines system monitor, incl. battery and overall GPU percentage; not just the VM data.

**Comment by dougiteixeira** [comment, marked as answer]
The integration is based on the [Proxmox API](https://pbs.proxmox.com/docs/api-viewer/index.html), so if this information exists in the API, it is possible to add it.

See if you have this information in your Proxmox interface...
